package.json

```diff
@@ -36,6 +36,7 @@
     "gulp-watch": "^4.3.11",
     "istanbul": "^0.4.5",
     "jsdoc-babel": "^0.2.1",
+    "jsdoc-baseline": "git+https://github.com/sugarcrm/jsdoc-baseline.git",
```

Comment:
Ah, I didn't see that you were recommending using Sugar's fork rather than the upstream version. My bad!

I got the template options to work using the upstream version. However, I have no objections to switching to Sugar's fork.